components/Common/Banner/index.module.scss

```diff
@@ -0,0 +1,28 @@
+.banner {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+  justify-content: center;
+  padding: 0.75rem 0;
+  text-align: center;
+  width: 100%;
+  @apply text-sm text-white underline;
+
+  svg {
+    height: 0.875rem;
```

Comment:
I bet there are tailwind variables for the intend height, no?

Reply:
it's SM i will fix that.